pulsar-common/src/main/java/org/apache/pulsar/common/naming/TopicName.java

```diff
@@ -320,6 +322,33 @@ public String getPersistenceNamingEncoding() {
         }
     }
 
+    public static TopicName fromPersistenceNamingEncoding(String name) throws Exception {
```

Comment:
I'm not sure what is this used for? Looks like we can parse it in the plugin and avoid introducing it into Pulsar?

Reply:
It's not for "one plugin", but maight be used for various plugins which may need to parse managed ledger name